Assets/Scripts/Models/Character.cs

```diff
@@ -161,6 +161,14 @@ void GetNewJob()
         // Get our destination from the job
         DestTile = myJob.tile;
 
+        // If the dest tile does not have neighbours it's very
+        if (DestTile.HasNeighboursOfType(TileType.Floor) == false)
```

Comment:
It should also search if there are Neighbours of tile type Ladder. Right now the Ladders can only be placed adjacent to Floors.